lib/puppet/settings/priority_setting.rb

```diff
@@ -6,11 +6,12 @@ class Puppet::Settings::PrioritySetting < Puppet::Settings::BaseSetting
   PRIORITY_MAP =
     if Puppet::Util::Platform.windows?
       require 'puppet/util/windows/process'
+      require 'puppet/util/ffi/ffi_constants'
       {
-        :high    => Puppet::Util::Windows::Process::HIGH_PRIORITY_CLASS,
-        :normal  => Puppet::Util::Windows::Process::NORMAL_PRIORITY_CLASS,
-        :low     => Puppet::Util::Windows::Process::BELOW_NORMAL_PRIORITY_CLASS,
-        :idle    => Puppet::Util::Windows::Process::IDLE_PRIORITY_CLASS
+        :high    => Puppet::Util::FFI::FFIConstants::HIGH_PRIORITY_CLASS,
+        :normal  => Puppet::Util::FFI::FFIConstants::NORMAL_PRIORITY_CLASS,
+        :low     => Puppet::Util::FFI::FFIConstants::BELOW_NORMAL_PRIORITY_CLASS,
+        :idle    => Puppet::Util::FFI::FFIConstants::IDLE_PRIORITY_CLASS
```

Comment:
I'm not a fan of the put everything in `Puppet::Util::*` namespace in general, but also because `Puppet::Util` is mixed into types and providers. Maybe `Puppet::FFI::Constants` instead? Since these  are windows specific, maybe `Puppet::FFI::WindowsConstants` or `Puppet::FFI::Windows::Constants`?

Reply:
I'm :+1: for `Puppet::FFI::Windows::Constants`